samples/nrf9160/nrf_cloud_agps/prj.conf

```diff
@@ -42,17 +41,25 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 # nRF Cloud
 CONFIG_CLOUD_API=y
 CONFIG_NRF_CLOUD=y
+CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y
 CONFIG_NRF_CLOUD_AGPS=y
-CONFIG_NRF_CLOUD_AGPS_LOG_LEVEL_INF=y
+CONFIG_NRF_CLOUD_AGPS_LOG_LEVEL_DBG=y
 CONFIG_MQTT_KEEPALIVE=1200
 CONFIG_MODEM_INFO=y
 
 # GPS
 CONFIG_NRF9160_GPS=y
 CONFIG_NRF9160_GPS_LOG_LEVEL_DBG=y
 
+# AGPS
+CONFIG_AGPS=y
+# CONFIG_AGPS_SRC_SUPL=y
+CONFIG_AGPS_LOG_LEVEL_DBG=y
+
 # Heap and stacks
 # Extended memory heap size needed for encoding nRF Cloud messages to JSON
 CONFIG_HEAP_MEM_POOL_SIZE=8192
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_NRF9160_GPS_THREAD_STACK_SIZE=6144
```

Comment:
That's a big stack. Perhaps the thread analyzer could be ran to check if it's required.

Reply:
Thread analyzer gave 4272 for nrf_cloud agps source and less for the SUPL client source. Changed it to 4608.